packages/dcos-integration-test/extra/conftest.py

```diff
@@ -195,6 +195,9 @@ def _wait_for_dcos(self):
         self._wait_for_srouter_slaves_endpoints()
         self._wait_for_metronome()
 
+    @retrying.retry(wait_fixed=2000,
```

Comment:
Sure, although none of the other wait functions here have an upper limit. But, it seems to be a corner case in advanced where auth isnt ready after leader election and adminrouter is.

What do you suggest? 5min? 2min?